### ISSUE TITLE
mat-select doesn't open. Only after other event. You can see the link in attachment.

### ISSUE THREAD

**Issue by bilenko-git** [angular/material2#9199]
#### Bug, feature request, or proposal:
Building of component on the fly.
https://www.youtube.com/watch?v=hLCWMniUszg

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
    "@angular/animations": "^4.4.6",
    "@angular/cdk": "^5.0.2",
    "@angular/common": "^4.3.6",
    "@angular/compiler": "^4.3.6",
    "@angular/core": "^4.3.6",
    "@angular/forms": "^4.3.6",
    "@angular/http": "^4.3.6",
    "@angular/material": "^5.0.2",
    "@angular/platform-browser": "^4.3.6",
    "@angular/platform-browser-dynamic": "^4.3.6",

#### Is there anything else we should know?
`@Component({
    selector: 'fr-input-select',
    template:
        <mat-form-field>
            <mat-select #mySelect class="fr-input-select"
                       [name]="md?.md?.name"
                       [placeholder]="md?.md?.caption"
                       [disabled]="disabled ? 'disabled': undefined"
                       [required]="validationRules?.required?.rule"
                       [(ngModel)]="selectedItem">
                <mat-option class="fr-input-select__disabled gray-color" [value]="">
                    {{md?.md?.caption}}
                    <i class="material-icons">&#xE5C7;</i>
                </mat-option>
                <mat-option
                        class="fr-input-select__items"
                        *ngFor="let item of md?.md?.items"
                        [value]="item">
                    {{item}}
                </mat-option>
            </mat-select>
        </mat-form-field>
        <validation [validationResult]="validationResult"></validation>
    `

**Comment by jelbourn** [comment]
Closing this issue as it does not follow the issue template. Please file a new issue that follows the template, which gives the team the information needed to investigate.

**Comment by VinMags16** [comment]
Just so you know you aren't alone I'm getting this error too.  I have a select with static data but it won't open until I do something else

**Comment by GaryChamberlain** [comment]
I'm getting this too, and I can't seem to create a simple repeatable example.  It's part of a very large application.
